R/derive_param_extreme_record.R

```diff
@@ -0,0 +1,233 @@
+#' Adds a Parameter based on first or last record from multiple sources
+#'
+#' Adds a Parameter based on first or last record from multiple sources
```

Comment:
Perhaps the second second isn't need, if the function is self-explanatory??

Reply:
![image](https://github.com/pharmaverse/admiral/assets/43018762/2d6cca7c-1796-41b8-8a0a-a2b40c49f7d1)
feels like our convention always seems to have both available, second can be a little more "descriptive" which I certainly think we can make slightly more clear

Reply:
Oh yeah - I think we should not just repeat the title in that section.  Perhaps another clean up task to give more context, assumption, etc in that section